tests/support/commands.ts

```diff
@@ -80,7 +80,9 @@ Cypress.Commands.add('createRuntimeConfig', ({ type } = {}): void => {
     );
   }
 
-  cy.findByLabelText(/object storage endpoint/i).type('http://0.0.0.0:9000');
+  cy.findByLabelText(/object storage .* endpoint\*/i).type(
```

Comment:
If you change this to 

```suggestion
  cy.findByLabelText(/^cloud object storage endpoint/i).type(
```

it should grab the appropriate element. `^` denotes the beginning of a string, which excludes the new `Public Cloud Object Storage Endpoint` element you've introduced in the UI by adding the new property in the schema. With this change in place the integration test passed in my local environment.

Reply:
Since [integration test execution](https://elyra.readthedocs.io/en/stable/developer_guide/contributing.html#ui-integration-tests) takes a long time for the entire suite to complete, I've  found it most efficient to do the following to only run the test/s I am interested in:
- run `make test-dependencies` to install the prerequisites
- create a new directory under `$REPO_HOME/tests`, e.g. `$REPO_HOME/tests/mytest`
- copy from `$REPO_HOME/tests/integration` the test file you want to run (here `$REPO_HOME/tests/integration/pipeline.ts`) to the new directory
- edit `$REPO_HOME/cypress.json` and replace the value of the `integrationFolder` property with the new directory name `$REPO_HOME/tests/mytest`
- run `make test-integration` will now only execute the tests in this file, which should take a lot less time

Maybe others have a more straight forward approach that's worth sharing?

Reply:
i commited your change already